packages/browser-sdk/src/lib/react/Grid/index.tsx

```diff
@@ -85,12 +134,10 @@ function Grid({ roomConnection, renderParticipant, videoGridGap = 0 }: GridProps
         const resizeObserver = new ResizeObserver(
             debounce(
                 () => {
-                    setContainerFrame(
-                        makeFrame({
-                            width: gridRef.current?.clientWidth,
-                            height: gridRef.current?.clientHeight,
-                        }),
-                    );
+                    setContainerBounds({
+                        width: gridRef.current?.clientWidth || 0,
```

Comment:
Perhaps we use non-zero values for the defaults here?

Reply:
Sure

Reply:
I put the default values to `width: 640` and `height: 480`